include/triton/Dialect/TritonGPU/IR/Dialect.h

```diff
@@ -83,8 +81,10 @@ bool isaDistributedLayout(Attribute layout);
 bool isSharedEncoding(Value value);
 
 } // namespace gpu
-} // namespace triton
 
+bool isExpensiveCat(CatOp cat, Attribute &targetEncoding);
```

Comment:
I think it should be placed under the gpu namespace as encodings are only available in TritonGPU dialect

Reply:
Done.